docs/04-operation-guides/troubleshooting/ac-01-application-gateway-troubleshooting.md

```diff
@@ -12,40 +12,25 @@ Follow these steps to detect the source of the problem:
 
 1. Check the logs.
 
-    Check the logs from the Application Gateway Pod to verify that the call reached Application Gateway.
-    To fetch these logs, run this command:
+Check the logs from the Application Gateway Pod to verify that the call reached Application Gateway.
+To fetch these logs, run this command:
+
+```bash
+kubectl -n kyma-system logs -l app=central-application-gateway -c central-application-gateway
+```
```

Comment:
``````suggestion
   Check the logs from the Application Gateway Pod to verify that the call reached Application Gateway.
   To fetch these logs, run this command:

   ```bash
   kubectl -n kyma-system logs -l app=central-application-gateway -c central-application-gateway
   ```
``````
These are supposed to be indented, as the code block belongs to step 1.